### PR TITLE
Add GitHub Workflow CI for Windows

### PR DESCRIPTION
**Pull request by swt2c**

### CODE REVIEW

File: .github/workflows/ci-windows.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+name: Windows CI
+on: [push, pull_request]
+jobs:
+  build:
+    runs-on: ubuntu-latest
```
Comment (jaymzh) [inline .github/workflows/ci-windows.yml:5]: why not just run on fedora latest directly?

Comment (swt2c) [reply .github/workflows/ci-windows.yml:5]: GitHub only provides Ubuntu as a virtual environment for Linux:
https://github.com/actions/virtual-environments

Comment (jaymzh) [reply .github/workflows/ci-windows.yml:5]: Oh right, I forgot about that. Doh.

**Comment by swt2c** [comment]
Ready for final review.

**Review by jaymzh — APPROVED** [review]
Thanks!

**Pull request merged**